src/common.browser/FileAudioSource.ts

```diff
@@ -56,6 +56,12 @@ export class FileAudioSource implements IAudioSource {
         return this.privAudioFormatPromise;
     }
 
+    public get blob(): Promise<Blob> {
+        const blobDeferral: Deferred<Blob> = new Deferred<Blob>();
+        blobDeferral.resolve(this.privFile);
+        return blobDeferral.promise();
```

Comment:
You should be able to use PromiseHelper.fromResult here.

Reply:
Corrected.